### ISSUE TITLE
Note input productivity improvement

### ISSUE THREAD

**Issue by moogly81** [musescore/MuseScore#16585]
### Your idea

This option could drastically improve productivity when entering notes with the computer keyboard. 

The option is simply : 
**make current note longer**

Let's assume : 
- 8th note is selected 
- normal note input mode is selected
- the shortcut **l** (small **L**, for "make note **L**onger") is assigned to this function

In this configuration, typing "l" would lengthen the current note by one 8th 

<img width="353" alt="image" src="https://user-images.githubusercontent.com/2691085/221818784-d30bae04-a7d1-4aff-8f95-5d7fa115f13a.png">

if 16th note is selected, then typing "l" would lenghten by a 16th note

### Problem to be solved

Note input on the keyboard is not very efficient.

**Comment by MarcSabatella** [comment]
It's an interesting idea, but for the record - the current system requires exactly the same number of keystrokes (11) in the first of your examples, and significantly fewer (12 vs 21) in the second.  So I can't see how it could be said to "drastically improve productivity".

For the record, there is already a command Shift+W that lengthens notes, but it also changes the selection on the toolbar, so it doesn't lead to the same series of keystrokes.  I suspect that people would not complain too much if it didn't change the toolbar selection though.

Also note there is the "foot pedal" real-time method that already works more or less exactly like this, except it works off MIDI input.

**Comment by moogly81** [comment]
Hi Marc, thanks for your answer, very much appreciated. 
I use W, Q, shift W, shift Q.  They are extremely useful but do not accomplish the stated goal of this feature request. 

The userfriendliness of the note input process boils down to personal preferences. Different strokes for different folks.
- When I type , I usually think **note pitch** first, **length** comes second. **Having to type only note names** (or 0 for silence), **and "l" feels simpler and more natural** to me. 
- Like you said, the "foot pedal" real-time method is very close to this proposition. But it's a whole new input method. Implementing this feature request would bring many of the real-time method advantages to the normal input mode. A very nice thing to this feature : it can be done without affecting anything else, so people who do not use it wouldn't even know it's there. 

### Additional remarks 
  - It should work with rests as well (type  **0ll** to insert 1,5 beats of rest)
  - the <shift L> shortcut could be assigned to the opposite function (make note shorter)

**Comment by Fristover** [comment]
I don't think this shortcut makes sense either, at most I would expect to get 2 tied notes (add a quarter, type "l", get 2 tied quarters) instead of just the longer value...

- To get 1,5 beats it's as easy as typing 0, 5 and ".", in this case it's the same number of strokes and in almost every other case it means less, it's also way more intuitive than what you're proposing.

To propose a feature, best thing you can do is making sure it's something that will be widely used by both new users and experienced ones (also feeling intuitive for both), this not really being the case for any of them, being rather just a personal preference
